### DISCUSSION TITLE
Light references for demo local installation

### DISCUSSION THREAD

**Discussion by PBMCHUGA** [broadinstitute/seqr#3196] [General]
Hello,

Is there a ligther subset of references data found here : 

gs://seqr-reference-data/GRCh37/all_reference_data
gs://seqr-reference-data/GRCh37/clinvar

Or can we access directly to the gs bucker from web browser ?

Would like to demonstrate seqr interest to my colleagues but have limited storage space for the moment

**Comment by hanars** [comment]
In order to demonstrate seqr, we recommend you register for free on [anvil](https://anvil.terra.bio/) and then you will be able to log into the Broad's [seqr](https://seqr.broadinstitute.org/) where you will have access to a Demo project
